.circleci/config.yml

```diff
@@ -293,6 +271,18 @@ jobs:
           name: Docker build
           command: |
             DOCKER_TAG=circleci make docker
+  build-chainlink-sgx:
+    machine: true
+    steps:
+      - checkout
+      - run:
+          name: Docker login
+          command: |
+            echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USER" --password-stdin
```

Comment:
Why do we pipe in the password rather than just specifying it via `-p`?

Reply:
Using literal passwords in Dockerfile is a big no-no, the Docker image produced then has the password encoded in the final image.